numpy/core/src/umath/umathmodule.c

```diff
@@ -273,7 +259,7 @@ intern_strings(void)
 /* Setup the umath module */
 /* Remove for time being, it is declared in __ufunc_api.h */
 /*static PyTypeObject PyUFunc_Type;*/
-
+/*
```

Comment:
I think this commented out section can now go, no?

Reply:
removed, also the next comment and an unneeded include